exonum/src/sandbox/sandbox_tests_helper.rs

```diff
@@ -48,6 +48,7 @@ pub const VALIDATOR_2: ValidatorId = ValidatorId(2);
 pub const VALIDATOR_3: ValidatorId = ValidatorId(3);
 pub const INCORRECT_VALIDATOR_ID: ValidatorId = ValidatorId(64_999);
 pub const PROPOSE_TIMEOUT: Milliseconds = 200;
+pub const DATA_SIZE: usize = 64;
```

Comment:
Maybe this constant should be in timestamping file since it is used only for timestamping txs.